### DISCUSSION TITLE
个人使用上的一些体验反馈

### DISCUSSION THREAD

**Discussion by RedRabbit1027** [Richasy/Bili.Uwp#1021] [想法]
目前本人在使用的是2.2204.4.0_x64版本。
相比从商店下载的版本，稳定性和优化上都感觉有非常明显的提升。
但是在具体使用的时候还是有些体验比较难受，在此给作者进行反馈一下。

1. 历史记录和稍后再看的位置塞进了用户头像，而用户头像做的太小，我用了三天才偶然意识到用户头像可以点开，里面有历史记录和稍后再看等功能，个人建议是将用户头像放回导航栏的最上面，然后加入导航栏定制显示栏目和下拉显示的功能。
2. 播放界面时，因为个人习惯将视频窗口以较小的分辨率放置在左下角的习惯，所以观看视频的时候，右边的相关视频和评论占用的位置就会变得格外多，所以希望加入宽屏模式？
3. 第三个应该算是有点微妙的一个疑问……科学上网的时候哔哩总是会直接显示“标签页详情数据请求异常”，是需要额外设置代理吗？

**Comment by Richasy** [comment]
1. 用户头像+折叠菜单是跟随 Microsoft Store 的设计，这在以后应该会成为 Win11 用户的操作习惯
2. 如果只需要放视频，打开视频的全窗口模式即可（F10）
3. 开启代理后，请按照 [常见问题](https://github.com/Richasy/Bili.Uwp/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98) 所述解除应用的本地网络环回
